modules/web/middleware/binding.go

```diff
@@ -136,7 +136,16 @@ func Validate(errs binding.Errors, data map[string]interface{}, f Form, l transl
 			case validation.ErrRegexPattern:
 				data["ErrorMsg"] = trName + l.Tr("form.regex_pattern_error", errs[0].Message)
 			default:
-				data["ErrorMsg"] = l.Tr("form.unknown_error") + " " + errs[0].Classification
+				msg := errs[0].Classification
+				if msg != "" {
+					msg += ": "
+				}
+
+				msg += errs[0].Message
```

Comment:
is it possible that errs[0].Message could be empty but errs[0].Classification is not?

Reply:
Maybe. I changed the code to
```go

				msg := errs[0].Classification
				if msg != "" && errs[0].Message != "" {
					msg += ": "
				}

				msg += errs[0].Message
				if msg == "" {
					msg = l.Tr("form.unknown_error")
				}
				data["ErrorMsg"] = trName + ": " + msg
```

This should now handle all cases correctly, no?